pages/matchs/matchs.module.css

```diff
@@ -0,0 +1,54 @@
+.container {
+  align-items: center;
+  justify-content: space-evenly;
+  margin: 100px auto;
+  width: 35%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: nowrap;
+  align-content: stretch;
+}
+
+.nameTeam {
+  background-color: white;
+  color: #333;
+  border-radius: 10px;
+  align-items: center;
+  text-align: center;
+  font-weight: bold;
+  margin-top: 10%;
+}
+
+.option {
+  background-color: white;
+  border-radius: 10px;
+  align-items: center;
+  text-align: center;
+  margin-top: 15%;
+  padding: 0.3rem;
+}
```

Comment:
Fix the aligment of these buttons to match with the figma design.

Reply:
Ready